src/components/Editor/Runners/HtmlRunner/HtmlRunner.test.js

```diff
@@ -50,50 +41,12 @@ describe("When page first loaded", () => {
     );
   });
 
-  test("iframe exists", () => {
-    const iframe = document.getElementsByClassName("htmlrunner-iframe")[0];
-    expect(iframe).toBeInTheDocument();
-  });
```

Comment:
⛏️  probably worth changing to check the iframe doesn't exist on load. Might even be worth adding a new test to check it does exist when `isEmbedded` is true

Reply:
ignore me... just seen the test below 😆